packages/sbcl/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://www.sbcl.org/
+TERMUX_PKG_DESCRIPTION="Steel Bank Common Lisp"
+TERMUX_PKG_LICENSE="BSD"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.3.0
+TERMUX_PKG_SRCURL=https://prdownloads.sourceforge.net/sbcl/sbcl-${TERMUX_PKG_VERSION}-source.tar.bz2
+TERMUX_PKG_BUILD_DEPENDS="ecl"
+TERMUX_PKG_SHA256=bf743949712ae02cb7493f3b8b57ce241948bf61131e36860ddb334da1439c97
+
+termux_step_make() {
+	cd $TERMUX_PKG_SRCDIR
+    XC_HOST="$TERMUX_PREFIX/bin/ecl --norc --c-stack 16777217"
```

Comment:
This won't work for cross-compilation, can't run $TERMUX_PREFIX/bin/ecl in the ubuntu docker image. Instead /usr/bin/ecl from ubuntu, or some ecl compiled in termux_step_host_build, needs to be used. 

Add ecl to [setup-ubuntu.sh](https://github.com/termux/termux-packages/blob/master/scripts/setup-ubuntu.sh) to install ubuntu's ecl in the docker image

Reply:
I suppose ubuntu 22.04's ecl will be too old as well, so we might need to compile a newer ecl in a `termux_step_host_build` step for this to work

Reply:
> we might need to compile a newer ecl in a termux_step_host_build step for this to work

In that case, it would be better to use SBCL as the host. I will try to do it!

I thought it would be possible to say "SBCL depends on ECL during build-time".